docker/tests/containerTestCommon.psm1

```diff
@@ -190,3 +190,72 @@ function Get-ContainerPowerShellVersion
     }
     return (Get-Content -Encoding Ascii $testContext.resolvedLogPath)[0]
 }
+
+# This function is used for basic validation of PS packages during a release;
+# During the process Docker files are filled out and executed with Docker build;
+# During the build PS packages are downloaded onto Docker containers, installed and selected Pester tests from PowerShell Github repo are executed.
+# This function must be run on a Docker host machine in 'Linux containers' mode, such as Windows 10 server with Hyper-V role installed.
+function ValidatePSPackage
```

Comment:
This seems to be a public function, should be named `Test-PSPackage`

Reply:
Renamed to Test-PSPackage